src/mapblock.cpp

```diff
@@ -382,7 +382,7 @@ void MapBlock::serialize(std::ostream &os_compressed, u8 version, bool disk, int
 		Bulk node data
 	*/
 	NameIdMapping nimap;
-	SharedBuffer<u8> buf;
+	Buffer<u8> buf;
```

Comment:
What's with this change?

Reply:
yes but also `Buffer` gains the ability to be cast into a string view